src/nvim/api/vim.c

```diff
@@ -1331,10 +1331,14 @@ Object nvim_load_context(Dictionary dict)
   return (Object)OBJECT_INIT;
 }
 
-/// Gets the current mode. |mode()|
-/// "blocking" is true if Nvim is waiting for input.
-///
-/// @returns Dictionary { "mode": String, "blocking": Boolean }
+/// Gets the current mode.
+///
+/// @returns Dictionary with items:
+///   - "mode": |mode()| name
+///   - "blocking": (Boolean) true if Nvim is waiting for input.
```

Comment:
Isn't it more "RPC requests are blocked while waiting for input" (the normal case is that Nvim processes RPC events _while_ waiting for input)